program/convert-feet-to-inches/ConvertFeetToInches.js

```diff
@@ -0,0 +1,3 @@
+let input=prompt("Enter a number");
+let result=parseInt(input)*12;
```

Comment:
Can you make the prompt label more appropriate  ?
`"Please enter the foot value"`

Reply:
Hmm! Sound good thanks